Core/Registry/Registry.cs

```diff
@@ -163,6 +163,18 @@ private void DeSerialisationFixes(StreamingContext context)
                 }
             }
 
+            // If we spot a default repo with the old .zip URL, flip it to the new .tar.gz URL
+            // Any other repo we leave *as-is*, even if it's the github meta-repo, as it's been
+            // custom-added by our user.
+
+            Repository default_repo;
+            var OldDefaultRepo = new Uri("https://github.com/KSP-CKAN/CKAN-meta/archive/master.zip");
```

Comment:
Local variables are typically always `camelCase`, so `defaultRepo` and `oldDefaultRepo`.